models/tasks.js

```diff
@@ -64,8 +64,33 @@ const fetchTask = async (taskId) => {
   }
 }
 
+/**
+ * Fetch all tasks of a user
+ *
+ * @return {Promise<tasks|Array>}
+ */
+
+const fetchUserTasks = async (username) => {
+  try {
+    const { user } = await fetchUser({ username })
+    const tasksSnapshot = await tasksModel.where('participants', 'array-contains', user.github_id).get()
```

Comment:
Here it should be `user.username` 
the participants will contain the RDS username.